packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -235,6 +236,15 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to 4, the appropriate elevation for app bars.
   final double elevation;
 
+  /// The material's shape as well its shadow.
```

Comment:
Explain what the shape means in terms of the app bar

Reply:
It's the shapeborder. 
I got the wrong name.

Reply:
After you've fixed things up and added a test, use git push to update this pull-request.